Makefile

```diff
@@ -1,46 +1,98 @@
 # Package configuration
 PROJECT = code-annotation
 COMMANDS = cli/server
-DEPENDENCIES = github.com/golang/dep/cmd/dep
+DEPENDENCIES = github.com/golang/dep/cmd/dep github.com/jteeuwen/go-bindata
 
 HOST ?= 127.0.0.1
 PORT ?= 8080
-SERVER_URL ?= //$(HOST):$(PORT)
+REACT_APP_SERVER_URL ?= //$(HOST):$(PORT) # frontend uses $(REACT_APP_SERVER_URL) as backend
+UI_DOMAIN ?= $(REACT_APP_SERVER_URL) # /oauth-callback redirects to $(UI_DOMAIN)/?token=__TOKEN__
+
+YARN_PRODUCTION ?= true
+
+# Tools
+YARN = yarn
+GODEP = dep
+GOLINT = golint
+GOVET = go vet
+BINDATA = go-bindata
+
+# ci variables
+TRAVIS_BUILD_DIR ?= $(shell pwd)
+CGO_ENABLED = 1
```

Comment:
why?

Reply:
at some point, it was needed, but I also needed to change a big amount of things to make the project deployable.... and maybe another change made not necessary the CGO.
Removed